packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -195,9 +195,13 @@ export const DetailsTile = ({
   hideShare,
   isPlaying,
   isPlayable = true,
+  isPlaylist = false,
+  isPublished = true,
+  onPressEdit = () => {},
```

Comment:
i personally dont provide default blank functions for things anymore, and just do `onPressEdit?.()` where appropriate.

Reply:
Good thought. will update